website/docs/components/icon.mdx

```diff
@@ -70,51 +102,90 @@ import { MdSettings } from "react-icons/md"
 // 2. Use the `as` prop to render the icon
 <Icon as={MdSettings} />
 
-// 3. Then you can pass all `Box` props to style the icon
-<Icon as={MdSettings} size="32px" color="green.400" />
+// 3. Then pass all props to style the icon
+<Icon as={MdSettings} boxSize="32px" color="teal.500" />
 ```
 
-Here's a working example:
+**Working example:**
 
 ```jsx
-<Icon as={MdSettings} size="32px" color="green.400" />
+<Icon as={MdSettings} boxSize="32px" color="teal.500" />
 ```
 
 ## Adding custom icons
 
-You can also create your own custom icons using the `createIcon` utility:
+Instead of using `Icon` component to render custom icons like this everytime,
+
+```jsx
+<Icon viewBox="0 0 200 200">
+  <path
+    fill="currentColor"
+    d="M 100, 100 m -75, 0 a 75,75 0 1,0 150,0 a 75,75 0 1,0 -150,0"
+  />
+</Icon>
+```
+
+Create custom icons using the `createIcon` utility and use them everywhere,
 
 ```jsx live=false
 import { createIcon } from "@chakra-ui/icon"
+
+// Using `path`
 export const UpDownIcon = createIcon({
   displayName: "UpDownIcon",
-  viewBox: "-1 -1 9 11",
-  d:
-    "M 3.5 0L 3.98809 -0.569442L 3.5 -0.987808L 3.01191 -0.569442L 3.5 0ZM 3.5 9L 3.01191 9.56944L 3.5 9.98781L 3.98809 9.56944L 3.5 9ZM 0.488094 3.56944L 3.98809 0.569442L 3.01191 -0.569442L -0.488094 2.43056L 0.488094 3.56944ZM 3.01191 0.569442L 6.51191 3.56944L 7.48809 2.43056L 3.98809 -0.569442L 3.01191 0.569442ZM -0.488094 6.56944L 3.01191 9.56944L 3.98809 8.43056L 0.488094 5.43056L -0.488094 6.56944ZM 3.98809 9.56944L 7.48809 6.56944L 6.51191 5.43056L 3.01191 8.43056L 3.98809 9.56944Z",
+  viewBox: "0 0 200 200",
+  path: (
+    <path
+      fill="currentColor"
+      d="M 100, 100 m -75, 0 a 75,75 0 1,0 150,0 a 75,75 0 1,0 -150,0"
+    />
+  ),
 })
-```
 
-The major advantage of `createIcon` is that it forwards ref properly.
+// OR Using `pathDefinition` directly
 
-If you use <Icon as={MdSettings}/> and `MdSettings` doesn't use `forwardRef`, the ref doesn't get forwarded automatically. This may report a React error in the console about passing ref to a function component. In short, you can use `<Icon as={MdSettings} />` if you know that `MdSettings` is forwarding the ref - otherwise, use the `createIcon` utility function and then call the icon directly: `<MdSettings />`.
```

Comment:
I think @sage merged this docs without verifying locally, got `MdSettings not defined` error with the above line.
I fixed it by removing this line moved the documentation above inside `Using an Icon library`

Reply:
My bad, thanks for fixing this.